docs/content/configuration/coordinator.md

```diff
@@ -102,3 +102,16 @@ To view last <n> entries of the audit history of coordinator dynamic config issu
 ```
 http://<COORDINATOR_IP>:<PORT>/druid/coordinator/v1/config/history?count=<n>
 ```
+
+
+# Lookups Dynamic Config (EXPERIMENTAL)
+Thse configuration options control the behavior of the Lookup dynamic configuration described in the [lookups page](../querying/lookups.html)
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.manager.lookups.hostDeleteTimeout`|How long to wait for a `DELETE` request to a particular node before considering the `DELETE` a failure|PT1s|
```

Comment:
I made it `druid.manager.lookups` to fit in with the other data manager configs in the coordinator:

```
            JsonConfigProvider.bind(binder, "druid.manager.segments", MetadataSegmentManagerConfig.class);
            JsonConfigProvider.bind(binder, "druid.manager.rules", MetadataRuleManagerConfig.class);
            JsonConfigProvider.bind(binder, "druid.manager.lookups", LookupCoordinatorManagerConfig.class);
```